Source/Plugins/OrientationAnalysis/OrientationAnalysisFilters/ReadCtfData.h

```diff
@@ -245,7 +253,7 @@ public slots:
   * @param m DataContainer instance pointer
   * @param tDims Tuple dimensions
   */
-  void readDataFile(CtfReader* reader, DataContainer::Pointer m, QVector<size_t>& tDims, CTF_READ_FLAG flag);
+  void readDataFile(CtfReader* reader, const DataContainer::Pointer &m, QVector<size_t>& tDims, CTF_READ_FLAG flag);
```

Comment:
Inconsistent use of `T &name` and `T& name` in the same line

Reply:
Thanks QtCreator.. Will get this fixed.